### PR TITLE
chore: add mohit1607

### PR DESCRIPTION
**Pull request by mohit1607**
Here is my data. I want to be a part of this community and want to contribute and learn.

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/880"></a>

### CODE REVIEW

File: public/data/mohit1607.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "MOHIT ASHLIYA",
```
Comment (syedsohan) [inline public/data/mohit1607.json:2]:
```suggestion
  "name": "Mohit Ashliya",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/mohit1607.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    {
+      "name":"Follow me on Twitter",
+      "url":"https://twitter.com/ashliya_mohit",
+      "icon":"twitter"
+    }
+  ],
+  "milestones":[
+    {
+      "title":"Started OpenSource Contribution",
```
Comment (syedsohan) [inline public/data/mohit1607.json:20]:
```suggestion
      "title":"Started Open Source Contribution",
```

File: public/data/mohit1607.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      "data":"December 2021",
+      "icon":"dollor",
+      "color":"grey",
+      "description":"From a day before 2023 I started my open-source journey."
```
Comment (syedsohan) [inline public/data/mohit1607.json:24]:
```suggestion
      "description":"From a day before 2023, I started my open source journey."
```
Maybe a the year `2023` is a typo so fix it.

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by syedsohan — CHANGES_REQUESTED** [review]
Change the PR title to - `fix: add mohit1607`

File: public/data/mohit1607.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "MOHIT ASHLIYA",
+  "type": "personal",
+  "bio": "Yo this is Superset for OpenSource Contribution.",
+  "avatar":"https://github.com/SuperSet1607.png",
+  "links": [
+    {
+      "name":"Follow me on Github",
+      "url":"https://github.com/mohit1607",
+      "icon":"github"
```
Comment (Panquesito7) [inline public/data/mohit1607.json:10]:
```suggestion
      "name": "Follow me on Github",
      "url": "https://github.com/mohit1607",
      "icon": "github"
```

File: public/data/mohit1607.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    {
+      "name":"Follow me on Twitter",
+      "url":"https://twitter.com/ashliya_mohit",
+      "icon":"twitter"
```
Comment (Panquesito7) [inline public/data/mohit1607.json:15]:
```suggestion
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/ashliya_mohit",
      "icon": "twitter"
```

File: public/data/mohit1607.json (hunk 3 above)
Comment (Panquesito7) [inline public/data/mohit1607.json:24]:
```suggestion
      "title": "Started OpenSource Contribution",
      "data": "December 2021",
      "icon": "dollor",
      "color": "grey",
      "description": "From a day before 2023 I started my open-source journey."
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/mohit1607.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "MOHIT ASHLIYA",
+  "type": "personal",
+  "bio": "Yo this is Superset for OpenSource Contribution.",
+  "avatar":"https://github.com/SuperSet1607.png",
```
Comment (syedsohan) [inline public/data/mohit1607.json:5]:
```suggestion
  "avatar": "https://github.com/SuperSet1607.png",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/mohit1607.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "twitter"
+    }
+  ],
+  "milestones":[
+    {
+      "title":"Started Open Source Contribution",
+      "data":"December 2021",
+      "icon":"dollor",
+      "color":"grey",
+      "description":"From a day before 2023, I started my open source journey."
```
Comment (syedsohan) [inline public/data/mohit1607.json:24]:
```suggestion
      "description": "From a day before 2023, I started my open source journey."
```

Is it `2023` or something else?

File: public/data/mohit1607.json [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ashliya_mohit",
+      "icon": "twitter"
+    }
+  ],
+  "milestones":[
+    {
+      "title":"Started Open Source Contribution",
```
Comment (syedsohan) [inline public/data/mohit1607.json:20]:
```suggestion
      "title": "Started Open Source Contribution",
```

File: public/data/mohit1607.json [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+      "data":"December 2021",
```
Comment (syedsohan) [inline public/data/mohit1607.json:21]:
```suggestion
      "data": "December 2021",
```

File: public/data/mohit1607.json [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+      "icon":"dollor",
```
Comment (syedsohan) [inline public/data/mohit1607.json:22]:
```suggestion
      "icon": "dollor",
```

File: public/data/mohit1607.json [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+      "color":"grey",
```
Comment (syedsohan) [inline public/data/mohit1607.json:23]:
```suggestion
      "color": "grey",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/mohit1607.json (hunk 7 above)
Comment (krishguptadev) [inline public/data/mohit1607.json:24]: I think you meant =>
```suggestion
      "description": "From a day before 2022, I started my open source journey."
```

File: public/data/mohit1607.json (hunk 10 above)
Comment (emmalearnscode) [inline public/data/mohit1607.json:22]: Change spelling to dollar

Comment (syedsohan) [inline public/data/mohit1607.json:22]:
```suggestion
      "icon": "dollar",
```

File: public/data/mohit1607.json [hunk 12]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Mohit Ashliya",
+  "type": "personal",
+  "bio": "Yo this is Superset for OpenSource Contribution.",
```
Comment (syedsohan) [inline public/data/mohit1607.json:4]:
```suggestion
  "bio": "Yo! this is Superset for Open Source Contribution.",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/mohit1607.json (hunk 10 above)
Comment (krishguptadev) [inline public/data/mohit1607.json:22]:
```suggestion
      "icon": "dollar",
```

File: public/data/mohit1607.json [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+  "avatar": "https://github.com/SuperSet1607.png",
```
Comment (Panquesito7) [inline public/data/mohit1607.json:5]: Wrong link.
```suggestion
  "avatar": "https://github.com/mohit1607.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/mohit1607.json [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "twitter"
+    }
+  ],
+  "milestones":[
+    {
+      "title": "Started Open Source Contribution",
+      "data": "December 2021",
+      "icon": "dollar",
+      "color":"grey",
+      "description": "From a day before 2023, I started my open source journey."
```
Comment (krishguptadev) [inline public/data/mohit1607.json:24]:
```suggestion
      "description": "From a day before 2022, I started my open source journey."
```

**Comment by emmalearnscode** [comment]
@mohit1607 do you need any help updating this pull request?

**Comment by emmalearnscode** [comment]
@mohit1607 ill close this pull request for now. Feel free to open it up again if you want to make the final changes.

**Pull request closed without merging**